src/client.rs

```diff
@@ -383,7 +399,7 @@ pub enum ClientFlowError {
     #[error(transparent)]
     Stream(#[from] StreamError),
     #[error("Expected `\\r\\n`, got `\\n`")]
-    ExpectedCrlfGotLf { discarded_bytes: Box<[u8]> },
+    ExpectedCrlfGotLf { discarded_bytes: Secret<Box<[u8]>> },
```

Comment:
Maybe it's better to implement a debug version by hand here. Would spare us some type conversions ...

Reply:
No, `Secret` is fine for me.